core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -487,7 +487,7 @@ class ControllerContext extends ControllerChannelContext {
     partitionLeadershipInfo.filter { case (topicPartition, leaderIsrAndControllerEpoch) =>
       !isTopicQueuedUpForDeletion(topicPartition.topic) &&
         leaderIsrAndControllerEpoch.leaderAndIsr.leader == brokerId &&
-        partitionReplicaAssignment(topicPartition).size > 1
+        partitionReplicaAssignment(topicPartition).lengthCompare(1) > 0
```

Comment:
FYI reviewers - 
`partitionReplicaAssignment(topicPartition)` is a scala `Seq`, hence, `lengthCompare()` is beneficial.

Reply:
The `lengthCompare` code is quite a bit less readable. Does it actually matter for the various code like this one? For example, if the Seq implementation is not `scala.List`, it won't make any difference aside from making the code less readable.